random/doc.go

```diff
@@ -0,0 +1,11 @@
+package random
```

Comment:
If you want to document the package:

```
// Package sort provides primitives for sorting slices and user-defined
// collections.
package sort
```

https://blog.golang.org/godoc

I think the following comments belong here